datasets/depmap/scripts/cell_line_processing.py

```diff
@@ -0,0 +1,156 @@
+import pandas as pd
+import re
+import os
+
+# configuration - change cell line and dependency cutoff as needed
+cell_line_name = "FADU"
+dependency_threshold = 0.5
+
+
+def check_cell_line(cell_line_name, omics_profiles, damaging_mutations_df, CRISPR_dependency, omics_expression, omics_cnv):
+    print(f"Checking cell line: {cell_line_name} presence in all datasets...")
+    match = omics_profiles[omics_profiles["StrippedCellLineName"].str.lower()== cell_line_name.lower()]
+
+
+    if match.empty:
+        print(f"Cell line '{cell_line_name}' not found in OmicsProfiles.")
+        return False, None
```

Comment:
Raising an error would be better than returning false if I understand the flow correctly.